lib/matplotlib/pyplot.py

```diff
@@ -2436,12 +2436,14 @@ def imsave(
 
 def matshow(A: ArrayLike, fignum: None | int = None, **kwargs) -> AxesImage:
     """
-    Display an array as a matrix in a new figure window.
-
-    The origin is set at the upper left hand corner and rows (first
-    dimension of the array) are displayed horizontally.  The aspect
-    ratio of the figure window is that of the array, unless this would
-    make an excessively short or narrow figure.
+    Display a 2D array as a matrix in a new figure window.
+
+    The origin is set at the upper left hand corner.
+    The indexing is `(row, column)`, where
```

Comment:
```suggestion
    The indexing is ``(row, column)``, where
```
Literal code in ReStructuredText requires double backticks.